cli/templatize.go

```diff
@@ -0,0 +1,412 @@
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"go.keploy.io/server/v2/config"
+	replaySvc "go.keploy.io/server/v2/pkg/service/replay"
+	"go.keploy.io/server/v2/utils"
+	"go.uber.org/zap"
+)
+
+func init() {
+	Register("Templatize", Templatize)
+}
+
+func Templatize(ctx context.Context, logger *zap.Logger, conf *config.Config, serviceFactory ServiceFactory, cmdConfigurator CmdConfigurator) *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:     "templatize",
+		Short:   "templatize the keploy testcases for re-record",
+		Example: `keploy templatize -c "/path/to/user/app"`,
```

Comment:
Improve the example, it should also give an example of adding different test-sets.